src/client/driver/command-executors/client-functions/replicator.ts

```diff
@@ -0,0 +1,132 @@
+import Replicator, { Transform } from 'replicator';
+import evalFunction from './eval-function';
+import {
+    NodeSnapshot,
+    ElementSnapshot,
+    ElementActionSnapshot,
+} from './selector-executor/node-snapshots';
+import { DomNodeClientFunctionResultError, UncaughtErrorInCustomDOMPropertyCode } from '../../../../shared/errors/index';
+import { CustomDOMProperties, SelectorDependencies } from './types';
+import adapter from './adapter/index';
+import selectorFilter from './selector-executor/filter';
+import { Dictionary } from '../../../../configuration/interfaces';
+
+
+const identity = (val: unknown): unknown => val;
+
+
+export function createReplicator (transforms: Transform[]): Replicator {
+    // NOTE: we will serialize replicator results
+    // to JSON with a command or command result.
+    // Therefore there is no need to do additional job here,
+    // so we use identity functions for serialization.
+    const replicator = new Replicator({
+        serialize:   identity,
+        deserialize: identity,
+    });
+
+    return replicator.addTransforms(transforms);
+}
+
+export class FunctionTransform implements Transform {
```

Comment:
Move each transform class to the separate file in the `/replicator/transforms` folder.